pulsenotify/templates/email_template.html

```diff
@@ -94,9 +94,7 @@
                         <!-- BEGIN FOOTER // -->
                         <em>Sent from pulse-notify at {{ date }}.</em>
                         <br />
-                        <strong>Problem? File a bug, or email</strong>
-                        <br />
-                        csheehan@mozilla.com
+                        <strong>Problem? File a bug on <a href="https://bugzilla.mozilla.org">Bugzilla</a>.
```

Comment:
Can you change the link to something like https://bugzilla.mozilla.org/enter_bug.cgi?product=Release%20Engineering&component=Release%20Automation